### DISCUSSION TITLE
Quality of life features

### DISCUSSION THREAD

**Discussion by UltraCenterHQ** [valinet/ExplorerPatcher#2743] [Ideas]
Add the feature to hover your folders into the programs list to add them into it (Just like StartAllBack)

Add Dynamic taskbar transparency

Add an option to use the Windows 11 "Icon hover" design (With the sound, battery, and wifi icons being selected together, instead of a gear icon)

**Comment by pyrates999** [comment]
Look at Open Shell.

**Reply by UltraCenterHQ** [reply]
Nah, Open Shell sucks now. It feels like they didn't even add any new features ever since I last used it years ago

**Reply by pyrates999** [reply]
Not if you use it for the windows 7 style start menu, which is still the best IMO.
